changelog/11780.txt

```diff
@@ -0,0 +1,5 @@
+````
```

Comment:
These extra indentations are not required. Please refer to other changelog entries for reference.